paper/paper.md

```diff
@@ -18,119 +18,121 @@ authors:
     orcid: 0000-0001-6083-7038
     equal-contrib: true # (This is how you can denote equal contributions between multiple authors)
     affiliation: 3
-  - name: Michael Schlottke-Lakemper
-    orcid: 0000-0002-3195-2536
-    affiliation: 1 # TODO: @Michael: Stuttgart or Augsburg
   - name: Gregor J. Gassner
     orcid: 0000-0002-1752-1158
     affiliation: 2
+  - name: Michael Schlottke-Lakemper
+    orcid: 0000-0002-3195-2536
+    affiliation: 1, 4
 affiliations:
  - name: High-Performance Computing Center Stuttgart, University of Stuttgart, Germany
    index: 1
  - name: Department of Mathematics and Computer Science, University of Cologne, Germany
    index: 2
  - name: Institute of Surface Science, Helmholtz-Zentrum hereon, Germany
    index: 3
-date: 19 April 2024 # TODO
+ - name: High-Performance Scientific Computing, University of Augsburg, Germany
+   index: 4
+date: \today
 bibliography: paper.bib
 ---
 
 # Summary
 
-TrixiParticles.jl, part of the Trixi Project [@schlottkelakemper2020trixi], is an innovative Julia-based open-source framework designed for particle-based multiphysics simulations.
-It aims to overcome the limitations of traditional mesh-based methods in handling complex geometries and specialized applications, such as computational fluid dynamics and structural mechanics,
-by offering a versatile platform for Smoothed Particle Hydrodynamics (SPH) and the Discrete Element Method (DEM), among others.
-Unique in its approach, TrixiParticles.jl facilitates the easy addition of new particle systems and their interactions, enhancing research flexibility.
-This is further complemented by its user-friendly design, which allows simulations to be configured directly with Julia code, eliminating the need for parameter files.
-This feature not only simplifies the integration of custom functionalities but also promotes rapid prototyping, establishing TrixiParticles.jl as a robust and adaptable tool for advancing multiphysics simulations in various scientific and engineering fields.
+TrixiParticles.jl, part of the Trixi Framework [@schlottkelakemper2020trixi], is an innovative Julia-based open-source software designed for particle-based multiphysics simulations.
+In this sense, we distinguish two approaches: Particles are either treated as physical particles or as mathematical interpolation points.
+The former case refers to methods that model separate, discrete particles with rotational degrees of freedom such as the Discrete Element Method (DEM) proposed by [@Cundall:1979].
+Whereas the latter case refers to the smoothed particle hydrodynamics (SPH) which is a numerical discretization method for solving continuum mechanic problems.
+SPH was originally developed by [@Monaghan:1977] to simulate astrophysical applications and is now widely used to simulate computational fluid dynamics, structural mechanics and even heat conduction problems.
```

Comment:
Does this fit in a summary? I agree that the previous version was a bit too much advertisement, but explaining that particles can either be physical particles or interpolation points seems a bit misplaced to me in the summary.